mmdet/core/hook/yolox_mode_switch_hook.py

```diff
@@ -32,7 +38,15 @@ def before_train_epoch(self, runner):
         if is_module_wrapper(model):
             model = model.module
         if (epoch + 1) == runner.max_epochs - self.num_last_epochs:
-            runner.logger.info('No mosaic and mixup aug now!')
-            train_loader.dataset.update_skip_type_keys(self.skip_type_keys)
             runner.logger.info('Add additional L1 loss now!')
             model.bbox_head.use_l1 = True
+
+        if self.is_multiprocess:
+            # Due to the multi-process feature, closing the augmentation
```

Comment:
Need to explain in detail why need to do this in comments, or add relevant links.